keps/prod-readiness/sig-network/3458.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3458
+beta:
+  approver: "@wojtek-t"
```

Comment:
The fact that I said I'm happy to take it doesn't mean it will be easy :)
See my comments below.